### PR TITLE
[MRG+1]Caught various exception, global scope of rawinput(python2 & 3 compatibility)

### PR DESCRIPTION
**Pull request by yash98**
- Exception for `os.chdir()` and `PushBullet(api_key)` caught and prompted user to rerun initialise

- checking device_id exists prevents exception to be thrown

- Global scope rawinput()
`if version_info[0] == 2:
    rawinput = raw_input
else:
    rawinput = input`

### CODE REVIEW

File: moboff.py [hunk 1]
```diff
@@ -66,7 +70,11 @@ def download(link, newdevice, video, delete):
             bold=True)
         quit()
 
-    os.chdir(directory)
+    try:
+        os.chdir(directory)
+    except OSError:
+        click.secho("The directory previously selected to download music can't be accessed. Please rerun moboff initialise.",)
```
Comment (Parth-Vader) [inline moboff.py]: remove the comma at the end.

File: moboff.py [hunk 2]
```diff
@@ -158,6 +167,10 @@ def initialise():
 
     device_id = int(rawinput()) - 1
 
+    if not 0<=device_id<=len(pb.devices):
+        click.secho("choose from available device")
```
Comment (Parth-Vader) [inline moboff.py]: Capitalise the starting letter, end with a full stop.

File: moboff.py [hunk 3]
```diff
@@ -110,7 +118,12 @@ def download(link, newdevice, video, delete):
 
     print("File to send : {0}".format(recent_download))
 
-    pb = Pushbullet(api_key)
+    try:
+        pb = Pushbullet(api_key)
+    except pushbullet.errors.InvalidKeyError:
+        click.secho("API key you previously entered is no longer valid. Please rerun moboff initialise")
```
Comment (Parth-Vader) [inline moboff.py]: end with a full stop.

**Comment by Parth-Vader** [comment]
Added some minor comments. After that, this can be merged.

**Comment by yash98** [comment]
I will make these changes and send another PR.

**Comment by Parth-Vader** [comment]
@yash98 Add commits to this PR only/

**Comment by yash98** [comment]
done.

**Pull request merged**